styles.scss

```diff
@@ -0,0 +1,299 @@
+@import 'variables';
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  background: $first-color;
+  font-family: coiny;
+}
+
+body {
+  .container {
+    margin: auto;
+    padding: 5px;
+    border: 3px solid $second-color;
+    border-radius: 10px;
+    width: 90%;
+    height: 95vh;
+    display: grid;
+    grid-template: 12% 79% 1fr / 100%;
+    grid-row-gap: 5px;
+    align-content: stretch;
+    align-items: stretch;
+
+    .header, .game-wrapper, .sentence-wrapper {
+      border: inherit;
+      border-radius: inherit;
+      background-color: $fourth-color;
+    }
+
+    .header {
+      .title {
+        text-align: center;
+        margin-top: .1em;
```

Comment:
are you sure what is `em` ? :P

Reply:
em is a relative type of measure, it refers to the parents element font size, i must have missed this one after i set it to this value, i just wanted it to be responsive, it seemed to work so... :) i'm open to any sugestions :) do you mean i need to change it to % or something?

Reply:
will discuss and explain, leave it for now ;)